packages/patternfly-4/content/design-guidelines/usage-and-behavior/lists-and-tables/design.md

```diff
@@ -2,14 +2,14 @@
 path: "/design-guidelines/usage-and-behavior/lists-and-tables"
 ---
 # Lists and tables
-PatternFly offers two components for displaying large data sets: the data list and the data table. While they satisfy similar use cases, choosing the correct component to use in your design will be dependent on the type of data you need to display.
+PatternFly offers two components for displaying large data sets: [data list](https://www.patternfly.org/v4/documentation/react/components/datalist/) and [table](https://www.patternfly.org/v4/documentation/react/components/table/). While they satisfy similar use cases, choosing the correct component to use in your design will be dependent on the type of data you need to display.
```

Comment:
we should use relative URLs here

Reply:
Happy to make that change. Just tell me what to put for the url. :-)